internal/infrastructure/manager.go

```diff
@@ -10,26 +11,30 @@ import (
 	"github.com/envoyproxy/gateway/internal/envoygateway"
 	"github.com/envoyproxy/gateway/internal/envoygateway/config"
 	"github.com/envoyproxy/gateway/internal/infrastructure/kubernetes"
+	"github.com/envoyproxy/gateway/internal/ir"
 )
 
+var _ Manager = (*kubernetes.Infra)(nil)
```

Comment:
why is this needed ?

Reply:
Just make sure kubernetes.Infra have implemented the interface.:)